src/views/RequestLinkForm/index.tsx

```diff
@@ -148,24 +149,25 @@ const RequestLinkForm = ({
   const [isSkipModalOpen, setSkipModalOpen] = useState<boolean>(false);
   const [isUnlinkModalOpen, setUnlinkModalOpen] = useState<boolean>(false);
 
-  const { data, error: relationError, loading: relationLoading } = useQuery<
```

Comment:
The generic query can still be used here.

I think there are also 2 requests happening with these new changes when it could still just be 1.